### ISSUE TITLE
UTF-8 is not well supported out-of-the-box.

### ISSUE THREAD

**Issue by efikarl** [onivim/oni2#2099]
Dear, developer

I'm a Chinese user of oni2, it should be a great editor in the future, but now I find an issue as title.

That's UTF-8 is not well supported, and CJK won't be rightly displayed. My configuration is as following:

{
  "editor.detectIndentation": true,
  "editor.fontFamily": "JetBrainsMono-Regular.ttf, Hack, Consolas, monospace",
  "editor.fontSize": 14,
  "editor.largeFileOptimizations": true,
  "editor.highlightActiveIndentGuide": true,
  "editor.indentSize": 4,
  "editor.insertSpaces": true,
  "editor.lineNumbers": "on",
  "editor.matchBrackets": true,
  "editor.minimap.enabled": true,
  "editor.minimap.maxColumn": 120,
  "editor.minimap.showSlider": true,
  "editor.renderIndentGuides": true,
  "editor.renderWhitespace": "none",
  "editor.rulers": [],
  "editor.tabSize": 4,
  "editor.zenMode.hideTabs": true,
  "editor.zenMode.singleFile": true,
  "files.exclude": ["_esy", "node_modules", ".git"],
  "files.encoding": "utf8",
  "workbench.activityBar.visible": true,
  "workbench.editor.showTabs": true,
  "workbench.iconTheme": "vs-seti",
  "workbench.sideBar.visible": true,
  "workbench.statusBar.visible": true,
  "workbench.tree.indent": 2,
  "vim.useSystemClipboard": ["yank"]
}

BR,
efikarl

**Comment by CrossR** [comment]
Closing as a duplicate of #868 and #1293.

There is some work ongoing in Revery at the moment on this, so hopefully soon we should have font fallback sorted so that this issue is fixed. The remaining UTF-8 issues should be sorted, but we can look into them more in detail once the font fallback is in place and using the editor is easier.
